test/fixtures/output/network/update-missing-flags.golden

```diff
@@ -0,0 +1 @@
+Error: at least one of the following flags must be set: `--name`
```

Comment:
Why is there only one flag listed here? Should we mark it as required instead?

Reply:
We allow updating only the name, I will mark it as required.